src/lib/y2network/widgets/wireless.rb

```diff
@@ -0,0 +1,91 @@
+require "y2network/widgets/wireless_essid"
+require "y2network/widgets/wireless_mode"
+require "y2network/dialogs/wireless_expert_settings"
+
+module Y2Network
+  module Widgets
+    # Top level widget for frame with general wireless settings
+    class Wireless < CWM::CustomWidget
+      attr_reader :settings
+
+      # @param settings [Y2Network::InterfaceConfigBuilder]
+      def initialize(settings)
+        textdomain "network"
+        @settings = settings
+      end
+
+      def contents
+        VBox(
+          VSpacing(0.5),
+          # Frame label
+          Frame(
+            _("Wireless Device Settings"),
```

Comment:
I've run the code and these widgets are centered. What about aligning them to the left?

Reply:
It keeps the pre-network-ng layout.

Reply:
Lets wait for review for feedback we get on new UI. I have no strong opinion on align.